arbeitszeit_flask/views/create_draft_view.py

```diff
@@ -73,32 +64,11 @@ def respond_to_get(self) -> Response:
         show user input form for plan draft.
```

Comment:
i guess it's a good example why comments are dangerous... and we can delete that whole comment.